index.rst

```diff
@@ -129,9 +128,10 @@ else than just writing your algorithm!
   Check out :doc:`Writing Bears <Developers/Writing_Bears>` for more
   information on this.
 
-Also, coala provides an external API using the dbus message protocol.
-This allows other applications to easily use the code analysis
-functionalities coala provides in their applications.
+To programmatically access coala's functionality, use ``coala-json`` or
```

Comment:
according to one user `programmatically` was a specific word he searched this page for